cup_of_join.py

```diff
@@ -0,0 +1,14 @@
+# Arguments :
+# args - Undefined number of lists
+# sep - random char
+# return value : list
+# This functions joins lists and between them appends the given char 'sep'
```

Comment:
change documentation to the write forma
![Capture](https://user-images.githubusercontent.com/13467749/159860073-a9935dec-4ad8-4332-b778-042ade683a51.JPG)
t